### PR TITLE
RELATED: RAIL-2812 Add 7.9.0 changelog entry

### PR DESCRIPTION
**Pull request by no23reason**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer:
- gdc-dashboards:

# Related Jira tasks

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/8228/ : SUCCESS in 9m 17s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4570/ : SUCCESS in 4s

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +27,11 @@ The REST API versions in the table are just for your information as the values a
 - Ranking filters that show the data from a set of the highest or lowest ranked values of an attribute based on the ranking criteria that you establish ([doc](https://sdk.gooddata.com/gooddata-ui/docs/ranking_filter_component.html))
 - The option to turning off data points in line charts, area charts, and combo charts ([doc](https://sdk.gooddata.com/gooddata-ui/docs/chart_config.html#configure-canvases))
 
+## 7.9.0
+
+### Added
+- Upgrade dependencies to their latest versions possible without breaking compatibility.
```
Comment (evgeniabogranskaya) [inline CHANGELOG.md:33]: The dependencies are upgraded to their latest possible versions to still be backward-compatible.

Comment (no23reason) [reply CHANGELOG.md]: Updated, thanks 🙂

**Comment by evgeniabogranskaya** [comment]
LGMT

**Review by kandl — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-react-components-gate-PR1783

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-fossa-scanning-zuul-el7/217/ : FAILURE in 2m 35s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-sonarqube-scan-zuul-el7/3/ : SUCCESS in 2m 47s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1997/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/4321/ : SUCCESS in 11m 57s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/8230/ : SUCCESS in 4m 54s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/3054/ : SUCCESS in 17m 19s

**Pull request merged**